migration/migrating-3-4/migrating-openshift-3-to-4.adoc

```diff
@@ -54,21 +54,21 @@ include::modules/migration-understanding-cam.adoc[leveloffset=+1]
 [id='Configuring-replication-repository_{context}']
 == Configuring a replication repository
 
-You must configure an S3-compatible replication repository. The CAM tool uses the replication repository as a temporary intermediate storage space for copying data from the source cluster to the target cluster, using the xref:migration-understanding-data-copy-methods_{context}[filesystem or snapshot data copy methods].
+Multi Cloud Gateway (MCG) is automatically installed as a replication repository. You can configure your own replication repository if you do not want to use the MCG repository.
+
+The CAM tool uses the replication repository as a temporary intermediate storage space for copying data from the source cluster to the target cluster, using the xref:migration-understanding-data-copy-methods_{context}[file system or snapshot copy method].
 
 The following storage providers are supported:
 
-// * xref:migration-configuring-noobaa_{context}[NooBaa] or other generic S3 storage provider (does not support snapshot copy method)
-* Multi Cloud Gateway (MCG) or other generic S3 storage provider
+* MCG or other generic S3 storage provider
 +
-MCG is installed by the CAM Operator and does not support the snapshot copy method.
+MCG, which is installed automatically on the same cluster as the CAM tool, does not support the snapshot copy method.
```

Comment:
Please remove the text on 'does not support the snapshot copy method.'
Snapshots are a feature of services providing volumes, block devices, etc, not for object storage.

A user who is using MCG for Object Store could still perform a snapshot on their PVs if the PV backing store supported snapshots.

but MCG/S3, etc is not involved in the snapshot piece.